src/flag_gems/__init__.py

```diff
@@ -147,6 +147,9 @@ def enable(lib=aten_lib):
     lib.impl("index_select", index_select, "CUDA")
     lib.impl("tile", tile, "CUDA")
     lib.impl("masked_fill", masked_fill, "CUDA")
+    lib.impl("masked_fill.Scalar", masked_fill, "CUDA")
+    lib.impl("masked_fill_", masked_fill_, "CUDA")
```

Comment:
there are no interfaces named 'masked_fill' and 'masked_fill_' in native_functions.yaml. I think we should delete line 149 and line 151.

Reply:
Modified them to *.Tensor